hack/verify-golint.sh

```diff
@@ -52,7 +52,7 @@ fi
 
 export IFS=$'\n'
 all_packages=(
-	$(go list -e ./... | egrep -v "/(third_party|vendor|staging|generated|clientset_generated)" | sed 's/k8s.io\/kubernetes\///g')
+	$(go list -e ./... | egrep -v "/(third_party|vendor|staging|generated|clientset_generated)" | sed -r "s/^_${KUBE_ROOT//\//\\/}\///g")
```

Comment:
Here's what i see on my mac - current directory is `/Users/dims/kubernetes/kubernetes`

```
[dims@dims-mac 18:01] ~/kubernetes/kubernetes ⟩ go list -e ./... | tail -5
_/Users/dims/kubernetes/kubernetes/vendor/gopkg.in/inf.v0
_/Users/dims/kubernetes/kubernetes/vendor/gopkg.in/natefinch/lumberjack.v2
_/Users/dims/kubernetes/kubernetes/vendor/gopkg.in/yaml.v2
_/Users/dims/kubernetes/kubernetes/vendor/k8s.io/heapster/metrics/api/v1/types
_/Users/dims/kubernetes/kubernetes/vendor/k8s.io/heapster/metrics/apis/metrics/v1alpha1
```

And on my ububtu - current directory is `/home/dims/kubernetes`

```
[dims@ubuntu-brix 19:19] ~/kubernetes ⟩ go list -e ./... | tail -5
_/home/dims/kubernetes/vendor/gopkg.in/inf.v0
_/home/dims/kubernetes/vendor/gopkg.in/natefinch/lumberjack.v2
_/home/dims/kubernetes/vendor/gopkg.in/yaml.v2
_/home/dims/kubernetes/vendor/k8s.io/heapster/metrics/api/v1/types
_/home/dims/kubernetes/vendor/k8s.io/heapster/metrics/apis/metrics/v1alpha1
```